DylibBinder/DBProperty.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SwiftReflector;
+using SwiftReflector.Inventory;
+
+namespace DylibBinder {
+	public class DBProperty {
+		public DBProperty (PropertyContents propertyContents)
+		{
+			Name = propertyContents.Name.Name;
+			IsStatic = propertyContents.Getter.IsStatic.ToString ();
+			Type = SwiftTypeToString.MapSwiftTypeToString (propertyContents.Getter.ReturnType, propertyContents.Class.ClassName.Module.Name);
+			GenericParameters = new DBGenericParameters (propertyContents.Getter.ReturnType);
+
+			Getter = new DBFunc (this, "Getter");
+			if (propertyContents.Setter != null) 
+				Setter = new DBFunc (this, "Setter");
+		}
+
+		public string Name { get; private set; }
+		public string IsStatic { get; }
+		public string Type { get; }
+		public DBFunc Getter { get; }
+		public DBFunc Setter { get; }
+		public DBGenericParameters GenericParameters { get; }
+
+		public string Accessibility { get; } = "Public";
```

Comment:
Candidate for an enum?

Reply:
I used to have an enum for this, but the only value I will be able to use is this "Public" so I got rid of it.

Reply:
Less secure or even redundant, if it is always the same value, why not using a constant.